<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewMySensors</class>
 <widget class="QDialog" name="DialogNewMySensors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditNodeid"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_nodeid_up">
       <property name="text">
        <string>Node ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_sensorid_up">
       <property name="text">
        <string>Sensor ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditSensorid"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_sensorid_down">
       <property name="text">
        <string>Sensor ID (Down)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_nodeid_down">
       <property name="text">
        <string>Node ID (Down)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditNodeidDown"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditSensoridDown"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gateway type</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="comboBoxSwitch">
       <item>
        <property name="text">
         <string>Normal switch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triple click switch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Long press switch</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBoxType">
       <item>
        <property name="text">
         <string>Serial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ethernet</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="check_smart">
       <property name="text">
        <string>Smart shutter</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QSpinBox" name="spin_time_up">
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="check_impulse">
       <property name="text">
        <string>Impulse shutter</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_time_up">
       <property name="text">
        <string>Up Duration (sec.)</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QSpinBox" name="spin_time_down">
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_time_down">
       <property name="text">
        <string>Down Duration (sec.)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEditTcpPort">
       <property name="text">
        <string>5003</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_nodeid_blue">
       <property name="text">
        <string>Node ID (Blue)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_sensorid_blue">
       <property name="text">
        <string>Sensor ID (Blue)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditNodeidBlue"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditSensoridBlue"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_gw_serialport">
       <property name="text">
        <string>Gateway serial port</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_gw_tcpport">
       <property name="text">
        <string>Gateway TCP port</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="comboBoxSerialport">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>/dev/ttyUSB0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyUSB1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyUSB2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyUSB3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyS0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyS1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyS2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyS3</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_gw_tcpip">
       <property name="text">
        <string>Gateway IP address</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lineEditTcpIP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_error_empty">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>146</red>
           <green>145</green>
           <blue>144</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>Please enter all required infos for this item..</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewMySensors</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
